stdlib/public/core/ArraySlice.swift

```diff
@@ -662,14 +662,14 @@ extension ArraySlice: RangeReplaceableCollection {
   /// `LazyMapCollection<Dictionary<String, Int>, Int>` to a simple
   /// `[String]`.
   ///
-  ///     func cacheImagesWithNames(names: [String]) {
+  ///     func cacheImages(withNames names: [String]) {
```

Comment:
@natecook1000 Do you know if we have establish precedent for "with" at the beginning of argument labels? My memory was that it's less common in Swift than in Objective-C, but the [API guidelines](https://swift.org/documentation/api-design-guidelines/#argument-labels) don't include an example like this.